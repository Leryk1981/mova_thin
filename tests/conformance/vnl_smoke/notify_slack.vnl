@budget_ms=5000
emit channel="slack" payload='{"target":"#ops","text":"Deploy done"}'

