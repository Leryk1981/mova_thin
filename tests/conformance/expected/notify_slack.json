{
  "mova_version": "3.3",
  "id": "notify_slack",
  "description": "Generated from VNL",
  "policies": {
    "timeout_ms": 15000,
    "budget_ms": 5000
  },
  "actions": [
    {
      "type": "emit_event",
      "channel": "slack",
      "payload": {
        "target": "#ops",
        "text": "Deploy done"
      }
    }
  ]
}